survey/2022/summary.md

```diff
@@ -0,0 +1,129 @@
+# Survey Summary and Takeaways
+
+This document summarizes the survey results and gives some insight into some
+takeaways from each category of the survey.
+
+In total, we had package manager maintainers (and/or reputable sources) from 11
+different package manager ecosystems responded to the survey. Including:
+
+-   PyPI (Python
+-   Gradle (JVM)
+-   Basalt  (Bash)
+-   Opam (OCaml)
+-   RubyGems (ruby)
+-   Npm (javascript)
+-   Pkg.go.dev (go)
+-   Cargo (Rust)
+-   Pub (Dart)
+-   Apache Maven
+-   Maven Central
```

Comment:
Are these both for Java/JVM packages?

Reply:
yep - adding (Java) and then changing grade to (Java) as well for consistency